src/common/containers/landing.css

```diff
@@ -42,6 +43,10 @@
   justify-content: space-between;
 }
 
+.row .rowImage {
+  padding: 0 1.25rem;
```

Comment:
Hmmm @bartaz I was re-thinking this change now coming back to it, I feel it would be better to put a ```20px``` padding left and right of the main container on the site. It shouldn't effect everything but it will put the desired guttering in and then solve the screen resize globally. Happy with this idea?